### DISCUSSION TITLE
Get authorization url error

### DISCUSSION THREAD

**Discussion by bguan2020** [nextauthjs/next-auth#2545] [Help]
I get this error when signing in, and there's no documentation on the error at this link: https://next-auth.js.org/errors#get_authorization_url_error.

Does anyone know what this error means? What url is having trouble?

**Comment by balazsorban44** [comment]
Please provide a reproduction/logs.

Also check the debug option: https://next-auth.js.org/configuration/options#debug

**Reply by bguan2020** [reply]
ok so I did this, here's what they give me: 
GET_AUTHORIZATION_URL_ERROR [ { statusCode: 401, data: 'Request token failed' } ]
SIGNIN_OAUTH_ERROR [ { statusCode: 401, data: 'Request token failed' } ]

Any idea what this means?

**Reply by balazsorban44** [reply]
Do you maybe have a reproduction, some code we could look at? What kind of provider, your next-auth config, your frontend code, how to trigger this error etc.?
